cmd/gbounty/bootstrap/helpers.go

```
@@ -33,6 +34,23 @@ func gbountyDir() (string, error) {
 	return filepath.Join(homedir, ".gbounty"), nil
 }
 
+func lastCheckFilePath() (string, error) {
+	dir, err := gbountyDir()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(dir, "last_check.txt"), nil
+}
+
+func updateLastCheckFile() error {
+	filePath, _ := lastCheckFilePath()
```

Please, don't ignore this error, but early return in such case! 🙏🏻